lib/elastic.js

```diff
@@ -11,6 +10,8 @@ var utils = require('./utils');
 var logger = require('juttle/lib/logger').getLogger('elastic');
 
 var instances;
+var ALREADY_CREATED_INDICES = {};
```

Comment:
mmm. So this is not really a constant. Generally in the adapter only constants are defined in all caps?